variables.tf

```diff
@@ -83,6 +83,19 @@ variable "logdna_tags" {
   default     = []
 }
 
+variable "logdna_access_tags" {
+  type        = list(string)
+  description = "A list of access tags to apply to the resources created by the module. For more information, see https://cloud.ibm.com/docs/account?topic=account-access-tags-tutorial."
```

Comment:
`to the resources created by the module` -> `to the LogDNA instance created by the module`